src/newlines.cpp

```diff
@@ -6594,7 +6598,8 @@ static void newlines_enum_entries(Chunk *open_brace, iarf_e av)
 
    Chunk *pc = Chunk::NullChunkPtr;
 
-   if (open_brace != nullptr)
+   if (  open_brace != nullptr
+      && open_brace->IsNotNullChunk())
```

Comment:
As some comments above, this is only a temporary check to prevent possible SEGV. No need to check for ```IsNotNullChunk()```